outer_region:
  delivery:
    points:
      - x: 1.3
        y: 0.7

      - x: 1.3
        y: -0.7

      - x: -0.6
        y: -0.7

      - x: -0.6
        y: 0.7


  table_dropping:
    points:
      - x: 0.1
        y: 0.1

      - x: 0.1
        y: -0.1

      - x: -0.1
        y: -0.1

      - x: -0.1
        y: 0.1

  table_picking:
    points:
      - x: 0.4
        y: 0.3

      - x: 0.4
        y: -0.3

      - x: -0.0
        y: -0.0

      - x: -0.0
        y: 0.0

  manual:
    points:
      - x: 0.7
        y: 0.3

      - x: 0.7
        y: -0.3

      - x: -0.3
        y: -0.3

      - x: -0.3
        y: 0.3

  error:
    points:
      - x: 0.7
        y: 0.3

      - x: 0.7
        y: -0.3

      - x: -0.0
        y: -0.0

      - x: -0.0
        y: 0.0

  suspend:
    points:
      - x: 0.7
        y: 0.3

      - x: 0.7
        y: -0.3

      - x: -0.3
        y: -0.3

      - x: -0.3
        y: 0.3

  none:
    points:
      - x: 0.7
        y: 0.3

      - x: 0.7
        y: -0.3

      - x: -0.3
        y: -0.3

      - x: -0.3
        y: 0.3

  charging:
    points:
      - x: 1.0
        y: 0.5

      - x: 1.0
        y: -0.5

      - x: -0.0
        y: -0.5

      - x: -0.0
        y: 0.5

  standby:
    points:
      - x: 1.0
        y: 0.5

      - x: 1.0
        y: -0.5

      - x: -0.4
        y: -0.5

      - x: -0.4
        y: 0.5


  fast:
    points:
      - x: 1.4    #1.4
        y: 0.5

      - x: 1.4    #1.4
        y: -0.5

      - x: -0.6
        y: -0.5

      - x: -0.6
        y: 0.5

  slow:
    points:
      - x: 1.1  #1.2
        y: 0.5

      - x: 1.1  #1.2
        y: -0.5

      - x: -0.5
        y: -0.5

      - x: -0.5
        y: 0.5

inner_region:
  points:
      - x: 1.4
        y: 0.3

      - x: 1.4
        y: -0.3

      - x: -0.4
        y: -0.3

      - x: -0.4
        y: 0.3

inner_region_top_speed:
  points:
      - x: 1.5
        y: 0.7    #.6

      - x: 1.5
        y: -0.7 #.6

      - x: 0.4  #4
        y: -0.7  #.6

      - x: 0.4   #4
        y: 0.7   #.6

inner_region_slow_speed:
  points:
      - x: 1.3
        y: 0.7   #.6

      - x: 1.3
        y: -0.7  #.6

      - x: 0.4  #4
        y: -0.7   #.6

      - x: 0.4  #4
        y: 0.7   #.6

front_region:
  points:
    - x: 1.3
      y: 0.4

    - x: 1.3
      y: -0.4

    - x: -0.0
      y: -0.0

    - x: -0.0
      y: 0.0

rear_region:
  points:
    - x: -0.8
      y: 0.4

    - x: -0.8
      y: -0.4

    - x: -0.3
      y: -0.4

    - x: -0.3
      y: 0.4

